src/types.py

```diff
@@ -54,10 +57,7 @@ def __init__(
             prefixes: list[str],
             exit_code: int = 0,
             **kwargs):
-        super().__init__(*args, **kwargs)
-        self.baba_loaded: bool = False
-        self.macros: dict = None
-        self.flags = None
+        ...
```

Comment:
Any reason why you're leaving this out?

Reply:
src/types is for stubs, i probably should move TilingMode back to constants

Reply:
Yeah I was wondering why you moved it. Maybe move it back then